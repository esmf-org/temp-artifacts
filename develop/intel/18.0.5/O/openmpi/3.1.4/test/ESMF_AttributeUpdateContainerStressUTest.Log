2022-07-20 14:33:35
20220720 150827.046 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220720 150827.046 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220720 150827.046 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220720 150827.046 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220720 150827.046 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220720 150827.046 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220720 150827.046 INFO             PET0 Running with ESMF Version   : v8.4.0b06-4-gd14c4971e8
20220720 150827.046 INFO             PET0 ESMF library build date/time: "Jul 20 2022" "14:28:33"
20220720 150827.046 INFO             PET0 ESMF library build location : /glade/scratch/dunlap/esmf-test-dev/intel_18.0.5_openmpi_O_develop/esmf
20220720 150827.046 INFO             PET0 ESMF_COMM                   : openmpi
20220720 150827.046 INFO             PET0 ESMF_MOAB                   : enabled
20220720 150827.046 INFO             PET0 ESMF_LAPACK                 : enabled
20220720 150827.046 INFO             PET0 ESMF_NETCDF                 : enabled
20220720 150827.046 INFO             PET0 ESMF_PNETCDF                : disabled
20220720 150827.046 INFO             PET0 ESMF_PIO                    : enabled
20220720 150827.046 INFO             PET0 ESMF_YAMLCPP                : enabled
20220720 150827.047 INFO             PET0  Beginning Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 792
20220720 150827.047 INFO             PET0  NUMBER_OF_PROCESSORS           4
20220720 150827.059 ERROR            PET0 ESMCI_Info.C:668 Info::erase() Not found  - [json.exception.out_of_range.403] key 'StandardName' not found
20220720 150827.059 ERROR            PET0 ESMCI_Info.C:688 Info::erase() Not found  - Internal subroutine call returned Error
20220720 150827.059 ERROR            PET0 ESMC_InfoCDef.C:243 ESMC_InfoErase() Not found  - Internal subroutine call returned Error
20220720 150827.059 ERROR            PET0 ESMF_Info.F90:2656 ESMF_InfoRemove() Not found  - Internal subroutine call returned Error
20220720 150827.059 ERROR            PET0 src/Superstructure/AttributeAPI/interface/ESMF_Attribute.F90:45812 ESMF_AttributeRemoveAttPackField Not found  - Internal subroutine call returned Error
20220720 150827.059 ERROR            PET0 ESMCI_Info.C:668 Info::erase() Not found  - [json.exception.out_of_range.403] key 'LongName' not found
20220720 150827.059 ERROR            PET0 ESMCI_Info.C:688 Info::erase() Not found  - Internal subroutine call returned Error
20220720 150827.059 ERROR            PET0 ESMC_InfoCDef.C:243 ESMC_InfoErase() Not found  - Internal subroutine call returned Error
20220720 150827.059 ERROR            PET0 ESMF_Info.F90:2656 ESMF_InfoRemove() Not found  - Internal subroutine call returned Error
20220720 150827.059 ERROR            PET0 src/Superstructure/AttributeAPI/interface/ESMF_Attribute.F90:45812 ESMF_AttributeRemoveAttPackField Not found  - Internal subroutine call returned Error
20220720 150827.059 ERROR            PET0 ESMCI_Info.C:668 Info::erase() Not found  - [json.exception.out_of_range.403] key 'Units' not found
20220720 150827.059 ERROR            PET0 ESMCI_Info.C:688 Info::erase() Not found  - Internal subroutine call returned Error
20220720 150827.059 ERROR            PET0 ESMC_InfoCDef.C:243 ESMC_InfoErase() Not found  - Internal subroutine call returned Error
20220720 150827.059 ERROR            PET0 ESMF_Info.F90:2656 ESMF_InfoRemove() Not found  - Internal subroutine call returned Error
20220720 150827.059 ERROR            PET0 src/Superstructure/AttributeAPI/interface/ESMF_Attribute.F90:45812 ESMF_AttributeRemoveAttPackField Not found  - Internal subroutine call returned Error
20220720 150827.063 INFO             PET0  PASS  Testing for correct value on field2, ESMF_AttributeUpdateContainerStressUTest.F90, line 893
20220720 150827.066 INFO             PET0  PASS  Testing for correct value on field3, ESMF_AttributeUpdateContainerStressUTest.F90, line 905
20220720 150827.066 INFO             PET0  PASS  Testing for correct value on field6, ESMF_AttributeUpdateContainerStressUTest.F90, line 917
20220720 150827.066 ERROR            PET0 src/Superstructure/AttributeAPI/interface/ESMF_Attribute.F90:43102 ESMF_AttributeGetAttPackField() Not found  - convention/purpose not found and attpack object present
20220720 150827.066 INFO             PET0  PASS  Testing for correct value on field7, ESMF_AttributeUpdateContainerStressUTest.F90, line 932
20220720 150827.066 INFO             PET0  PASS  Testing for correct value on field8, ESMF_AttributeUpdateContainerStressUTest.F90, line 945
20220720 150827.066 INFO             PET0  Ending Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 974
20220720 150827.066 INFO             PET0 Finalizing ESMF
20220720 150827.046 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220720 150827.046 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220720 150827.046 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220720 150827.046 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220720 150827.046 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220720 150827.046 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220720 150827.046 INFO             PET1 Running with ESMF Version   : v8.4.0b06-4-gd14c4971e8
20220720 150827.046 INFO             PET1 ESMF library build date/time: "Jul 20 2022" "14:28:33"
20220720 150827.046 INFO             PET1 ESMF library build location : /glade/scratch/dunlap/esmf-test-dev/intel_18.0.5_openmpi_O_develop/esmf
20220720 150827.046 INFO             PET1 ESMF_COMM                   : openmpi
20220720 150827.046 INFO             PET1 ESMF_MOAB                   : enabled
20220720 150827.046 INFO             PET1 ESMF_LAPACK                 : enabled
20220720 150827.046 INFO             PET1 ESMF_NETCDF                 : enabled
20220720 150827.046 INFO             PET1 ESMF_PNETCDF                : disabled
20220720 150827.046 INFO             PET1 ESMF_PIO                    : enabled
20220720 150827.046 INFO             PET1 ESMF_YAMLCPP                : enabled
20220720 150827.047 INFO             PET1  Beginning Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 792
20220720 150827.047 INFO             PET1  NUMBER_OF_PROCESSORS           4
20220720 150827.059 ERROR            PET1 ESMCI_Info.C:668 Info::erase() Not found  - [json.exception.out_of_range.403] key 'StandardName' not found
20220720 150827.059 ERROR            PET1 ESMCI_Info.C:688 Info::erase() Not found  - Internal subroutine call returned Error
20220720 150827.059 ERROR            PET1 ESMC_InfoCDef.C:243 ESMC_InfoErase() Not found  - Internal subroutine call returned Error
20220720 150827.059 ERROR            PET1 ESMF_Info.F90:2656 ESMF_InfoRemove() Not found  - Internal subroutine call returned Error
20220720 150827.059 ERROR            PET1 src/Superstructure/AttributeAPI/interface/ESMF_Attribute.F90:45812 ESMF_AttributeRemoveAttPackField Not found  - Internal subroutine call returned Error
20220720 150827.059 ERROR            PET1 ESMCI_Info.C:668 Info::erase() Not found  - [json.exception.out_of_range.403] key 'LongName' not found
20220720 150827.059 ERROR            PET1 ESMCI_Info.C:688 Info::erase() Not found  - Internal subroutine call returned Error
20220720 150827.059 ERROR            PET1 ESMC_InfoCDef.C:243 ESMC_InfoErase() Not found  - Internal subroutine call returned Error
20220720 150827.059 ERROR            PET1 ESMF_Info.F90:2656 ESMF_InfoRemove() Not found  - Internal subroutine call returned Error
20220720 150827.059 ERROR            PET1 src/Superstructure/AttributeAPI/interface/ESMF_Attribute.F90:45812 ESMF_AttributeRemoveAttPackField Not found  - Internal subroutine call returned Error
20220720 150827.059 ERROR            PET1 ESMCI_Info.C:668 Info::erase() Not found  - [json.exception.out_of_range.403] key 'Units' not found
20220720 150827.059 ERROR            PET1 ESMCI_Info.C:688 Info::erase() Not found  - Internal subroutine call returned Error
20220720 150827.059 ERROR            PET1 ESMC_InfoCDef.C:243 ESMC_InfoErase() Not found  - Internal subroutine call returned Error
20220720 150827.059 ERROR            PET1 ESMF_Info.F90:2656 ESMF_InfoRemove() Not found  - Internal subroutine call returned Error
20220720 150827.059 ERROR            PET1 src/Superstructure/AttributeAPI/interface/ESMF_Attribute.F90:45812 ESMF_AttributeRemoveAttPackField Not found  - Internal subroutine call returned Error
20220720 150827.063 INFO             PET1  PASS  Testing for correct value on field2, ESMF_AttributeUpdateContainerStressUTest.F90, line 893
20220720 150827.066 INFO             PET1  PASS  Testing for correct value on field3, ESMF_AttributeUpdateContainerStressUTest.F90, line 905
20220720 150827.066 INFO             PET1  PASS  Testing for correct value on field6, ESMF_AttributeUpdateContainerStressUTest.F90, line 917
20220720 150827.066 ERROR            PET1 src/Superstructure/AttributeAPI/interface/ESMF_Attribute.F90:43102 ESMF_AttributeGetAttPackField() Not found  - convention/purpose not found and attpack object present
20220720 150827.066 INFO             PET1  PASS  Testing for correct value on field7, ESMF_AttributeUpdateContainerStressUTest.F90, line 932
20220720 150827.066 INFO             PET1  PASS  Testing for correct value on field8, ESMF_AttributeUpdateContainerStressUTest.F90, line 945
20220720 150827.066 INFO             PET1  Ending Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 974
20220720 150827.066 INFO             PET1 Finalizing ESMF
20220720 150827.046 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220720 150827.046 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220720 150827.046 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220720 150827.046 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220720 150827.046 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220720 150827.046 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220720 150827.046 INFO             PET2 Running with ESMF Version   : v8.4.0b06-4-gd14c4971e8
20220720 150827.046 INFO             PET2 ESMF library build date/time: "Jul 20 2022" "14:28:33"
20220720 150827.046 INFO             PET2 ESMF library build location : /glade/scratch/dunlap/esmf-test-dev/intel_18.0.5_openmpi_O_develop/esmf
20220720 150827.046 INFO             PET2 ESMF_COMM                   : openmpi
20220720 150827.046 INFO             PET2 ESMF_MOAB                   : enabled
20220720 150827.046 INFO             PET2 ESMF_LAPACK                 : enabled
20220720 150827.046 INFO             PET2 ESMF_NETCDF                 : enabled
20220720 150827.046 INFO             PET2 ESMF_PNETCDF                : disabled
20220720 150827.046 INFO             PET2 ESMF_PIO                    : enabled
20220720 150827.046 INFO             PET2 ESMF_YAMLCPP                : enabled
20220720 150827.047 INFO             PET2  Beginning Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 792
20220720 150827.047 INFO             PET2  NUMBER_OF_PROCESSORS           4
20220720 150827.063 INFO             PET2  PASS  Testing for correct value on field2, ESMF_AttributeUpdateContainerStressUTest.F90, line 893
20220720 150827.063 INFO             PET2  PASS  Testing for correct value on field3, ESMF_AttributeUpdateContainerStressUTest.F90, line 905
20220720 150827.063 INFO             PET2  PASS  Testing for correct value on field6, ESMF_AttributeUpdateContainerStressUTest.F90, line 917
20220720 150827.063 ERROR            PET2 src/Superstructure/AttributeAPI/interface/ESMF_Attribute.F90:43102 ESMF_AttributeGetAttPackField() Not found  - convention/purpose not found and attpack object present
20220720 150827.063 INFO             PET2  PASS  Testing for correct value on field7, ESMF_AttributeUpdateContainerStressUTest.F90, line 932
20220720 150827.063 INFO             PET2  PASS  Testing for correct value on field8, ESMF_AttributeUpdateContainerStressUTest.F90, line 945
20220720 150827.063 INFO             PET2  Ending Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 974
20220720 150827.063 INFO             PET2 Finalizing ESMF
20220720 150827.046 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220720 150827.046 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220720 150827.046 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220720 150827.046 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220720 150827.046 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220720 150827.046 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220720 150827.046 INFO             PET3 Running with ESMF Version   : v8.4.0b06-4-gd14c4971e8
20220720 150827.046 INFO             PET3 ESMF library build date/time: "Jul 20 2022" "14:28:33"
20220720 150827.046 INFO             PET3 ESMF library build location : /glade/scratch/dunlap/esmf-test-dev/intel_18.0.5_openmpi_O_develop/esmf
20220720 150827.046 INFO             PET3 ESMF_COMM                   : openmpi
20220720 150827.046 INFO             PET3 ESMF_MOAB                   : enabled
20220720 150827.046 INFO             PET3 ESMF_LAPACK                 : enabled
20220720 150827.046 INFO             PET3 ESMF_NETCDF                 : enabled
20220720 150827.046 INFO             PET3 ESMF_PNETCDF                : disabled
20220720 150827.046 INFO             PET3 ESMF_PIO                    : enabled
20220720 150827.046 INFO             PET3 ESMF_YAMLCPP                : enabled
20220720 150827.047 INFO             PET3  Beginning Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 792
20220720 150827.047 INFO             PET3  NUMBER_OF_PROCESSORS           4
20220720 150827.063 INFO             PET3  PASS  Testing for correct value on field2, ESMF_AttributeUpdateContainerStressUTest.F90, line 893
20220720 150827.063 INFO             PET3  PASS  Testing for correct value on field3, ESMF_AttributeUpdateContainerStressUTest.F90, line 905
20220720 150827.063 INFO             PET3  PASS  Testing for correct value on field6, ESMF_AttributeUpdateContainerStressUTest.F90, line 917
20220720 150827.063 ERROR            PET3 src/Superstructure/AttributeAPI/interface/ESMF_Attribute.F90:43102 ESMF_AttributeGetAttPackField() Not found  - convention/purpose not found and attpack object present
20220720 150827.063 INFO             PET3  PASS  Testing for correct value on field7, ESMF_AttributeUpdateContainerStressUTest.F90, line 932
20220720 150827.063 INFO             PET3  PASS  Testing for correct value on field8, ESMF_AttributeUpdateContainerStressUTest.F90, line 945
20220720 150827.064 INFO             PET3  Ending Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 974
20220720 150827.064 INFO             PET3 Finalizing ESMF
