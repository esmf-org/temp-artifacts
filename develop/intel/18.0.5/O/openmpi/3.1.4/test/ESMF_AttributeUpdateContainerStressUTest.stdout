2022-07-20 14:33:35
  Beginning Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 792
  Beginning Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 792
  Beginning Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 792
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 792
  NUMBER_OF_PROCESSORS           4
  NUMBER_OF_PROCESSORS           4
  NUMBER_OF_PROCESSORS           4
  PASS  Testing for correct value on field2, ESMF_AttributeUpdateContainerStress
 UTest.F90, line 893
  PASS  Testing for correct value on field2, ESMF_AttributeUpdateContainerStress
 UTest.F90, line 893
  PASS  Testing for correct value on field2, ESMF_AttributeUpdateContainerStress
 UTest.F90, line 893
  PASS  Testing for correct value on field3, ESMF_AttributeUpdateContainerStress
 UTest.F90, line 905
  PASS  Testing for correct value on field6, ESMF_AttributeUpdateContainerStress
 UTest.F90, line 917
  PASS  Testing for correct value on field2, ESMF_AttributeUpdateContainerStress
 UTest.F90, line 893
  PASS  Testing for correct value on field7, ESMF_AttributeUpdateContainerStress
 UTest.F90, line 932
  PASS  Testing for correct value on field8, ESMF_AttributeUpdateContainerStress
 UTest.F90, line 945
  PASS  Testing for correct value on field3, ESMF_AttributeUpdateContainerStress
 UTest.F90, line 905
  PASS  Testing for correct value on field6, ESMF_AttributeUpdateContainerStress
 UTest.F90, line 917
  Ending Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 974
  PASS  Testing for correct value on field7, ESMF_AttributeUpdateContainerStress
 UTest.F90, line 932
  PASS  Testing for correct value on field8, ESMF_AttributeUpdateContainerStress
 UTest.F90, line 945
  Ending Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 974
  PASS  Testing for correct value on field3, ESMF_AttributeUpdateContainerStress
 UTest.F90, line 905
  PASS  Testing for correct value on field6, ESMF_AttributeUpdateContainerStress
 UTest.F90, line 917
  PASS  Testing for correct value on field3, ESMF_AttributeUpdateContainerStress
 UTest.F90, line 905
  PASS  Testing for correct value on field7, ESMF_AttributeUpdateContainerStress
 UTest.F90, line 932
  PASS  Testing for correct value on field6, ESMF_AttributeUpdateContainerStress
 UTest.F90, line 917
  PASS  Testing for correct value on field8, ESMF_AttributeUpdateContainerStress
 UTest.F90, line 945
  PASS  Testing for correct value on field7, ESMF_AttributeUpdateContainerStress
 UTest.F90, line 932
  Ending Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 974
  PASS  Testing for correct value on field8, ESMF_AttributeUpdateContainerStress
 UTest.F90, line 945
  Ending Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 974
  PET           2  Test Elapsed Time    24.0280000000000       msec.
  PET           3  Test Elapsed Time    23.8020000000000       msec.
  PET           0  Test Elapsed Time    24.0030000000000       msec.
  PET           1  Test Elapsed Time    25.4550000000000       msec.
